lib/seattleflu/id3c/cli/command/etl/redcap_det_fh_airs.py

```diff
@@ -0,0 +1,910 @@
+        # The study documents say: 
+        #    If they report symptoms on a survey, 
+        #    they will be prompted to self-collect a nasal swab,
+        #    which will be delivered and returned via courier, 
+        #    and complete a brief symptom survey
+        # which doesn't even rhyme.
+        elif redcap_record_instance.event_name in ENCOUNTER_EVENT_NAMES:
+            if is_complete('weekly', redcap_record_instance):
+            #TODO: Do we skip events when no kit was sent, or are they still relevant?
+            #and \
+            #    (is_complete('airs_kit_activation_complete', redcap_record_instance) or \
+            #     is_complete('airs_kit_activation_2_complete', redcap_record_instance)):
+                event_type = EventType.ENCOUNTER
+                collection_method = CollectionMethod.SWAB_AND_SEND
+            else:
+                LOG.debug("Skipping record id: "
+                    f"{redcap_record_instance.get('subject_id')}, "
+                    "encounter: "
+                    f"{redcap_record_instance.get('event_name')}: "
+                    "insufficient information to construct encounter")
+                continue
+        else:
+            LOG.info(f"Skipping event: {redcap_record_instance.event_name!r} for record "
+            f"{redcap_record_instance.get('subject_id')} because the event is not one "
+            "that we process")
+            continue
+
+        # Skip an ENCOUNTER instance if we don't have the data we need to
+        # create an encounter.
+        # In this study they may send up to 2 AIRS kits per weekly encounter,
+        #  so I guess we need to check both
+        # TODO: verify with study that this is irrelevant if we only check if 'weekly' is complete, above.
+        if event_type == EventType.ENCOUNTER:
+            if not is_complete('weekly', redcap_record_instance) \
+                or not collection_method:
+                    LOG.debug("Skipping record id: "
+                        f"{redcap_record_instance.get('subject_id')}, "
+                        " encounter: "
+                        f"{redcap_record_instance.get('event_name')}, "
+                        ": insufficient information to construct encounter")
+                    continue
+
+        # We should never have to consult this
+        location_site_map = {
+            'bothell':  'UWBothell',
+            'odegaard': 'UWOdegaardLibrary',
+            'slu':      'UWSouthLakeUnion',
+            'tacoma':   'UWTacoma',
+            'uw_club':  'UWClub'
+            }
+
+        # TODO: since there's only one collection method
+        #  (courier-delivered swabs) I believe this will always be the same.
+        site_reference = create_site_reference(
+            location = None,
+            site_map = location_site_map,
+            default_site = SWAB_AND_SEND_SITE,
+            system_identifier = INTERNAL_SYSTEM)
```

Comment:
These look HCT specific. I think you're correct, that since these are always swab and send, this can be simplified.

Reply:
I've removed the `location_map` and set `site_map` to None because it's always swab-and-send. That will trigger `create_site_reference()` to use the specified `default_site`.